FreeRTOS-Plus/Source/FreeRTOS-Plus-TCP/FreeRTOS_IP.c

```diff
@@ -2379,6 +2384,11 @@ BaseType_t location = 0;
 		( usChecksum == ipINVALID_LENGTH ) )
 	{
 		FreeRTOS_printf( ( "CRC error: %04x location %ld\n", usChecksum, location ) );
+		
+		/* If FreeRTOS_printf is not defined, not using xLocation will be a violation of MISRA
```

Comment:
Can you change "xLocation" to be "location" to match the variable name?

Reply:
I was supposed to fix that in the [previous commit](https://github.com/FreeRTOS/FreeRTOS/pull/230/commits/a3e02c19656da2732445356bc5bd45a0601f5b7b). Thanks for spotting that mistake.
It is fixed now.